src/Microsoft.Diagnostics.Monitoring.WebApi/ProfilerMessage.cs

```diff
@@ -7,28 +7,48 @@
 
 namespace Microsoft.Diagnostics.Monitoring
 {
-    internal enum IpcCommand : short
+    internal enum CommandSet : ushort
+    {
+        DotnetMonitor,
```

Comment:
nit: `ServerResponse` to be clear what the intended use of the commands within the command set.